nodes/persistence-node/src/services/IpfsGatewayApi.ts

```diff
@@ -104,6 +105,15 @@ export class IpfsGatewayApi {
       });
     }));
 
+    app.get('/api/v0/pin/ls', handleError(async (req, res) => {
```

Comment:
since this is our custom implementation and not the IPFS node's pins, it should probably be in a custom endpoint. Maybe `/pin/ls` ?

Reply:
Fixed.